txpool/txpool_test.go

```diff
@@ -374,13 +375,16 @@ func TestTxnQueue_Heap(t *testing.T) {
 	})
 }
 
-func generateTx(from types.Address, value *big.Int) *types.Transaction {
+func generateTx(from types.Address, value *big.Int, size uint64) *types.Transaction {
+	data := make([]byte, size)
+	rand.Read(data)
```

Comment:
`generateTx` is supposed to be a general purpose method for transaction generation. If tomorrow we have a use-case where we want to add actual input to this helper method, it would render it useless since you've overwritten the `Input` storage with your predefined input within the method.

I suggest 2 options for fixing this:
1. Add an `Input` field as a parameter which will be passed to the transaction `Input` field.
2. Leave the function the way it was, but set the `Input` field from within your test case. This way, you save the additional parameter pass for tests which don't even utilize the `Input` field, but use this method

Reply:
That makes sense, will do the required modifications. Thanks @zivkovicmilos

Reply:
Fixed it. Here's the commit-[link](https://github.com/0xPolygon/polygon-sdk/pull/155/commits/ea46815256cb0fa65352cf667ec28b87c1e87c40)